src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -2021,7 +2021,7 @@ export default class Aircraft {
             }
 
             // lock ILS if at the right angle and altitude
-            if ((abs(this.altitude - glideslope_altitude) < glideslope_window)
+            if (this.altitude < (glideslope_altitude + glideslope_window)
                 && (abs(offset_angle) < degreesToRadians(10))
                 && (offset[1] < ils)
```

Comment:
I like where your head is at!  I just wanted to call that out for future work.  These will probably need to be tackled as-we-come-to-them.